test/testing/test_statistics_generation.py

```diff
@@ -58,7 +57,7 @@ def teardown_class(cls):
     @pytest.mark.integration
     def test_metrics_single_wrapper_runs_successfully_and_outfile_exists(self):
         # reset the tensorflow graph for new training
-        tf.reset_default_graph()
+        #tf.reset_default_graph()
```

Comment:
Just curious: why is it commented and not deleted?

Reply:
Hmm I can - I just kept them because, I'm not quite sure what some of these functionalities will be at all moving forward; they seem to track the training evolution (or one of them does), so I wanted to keep these here as a reminder that more work might actually need to be done or we might scrap these functions altogether. I can just delete them here though if you'd like.

Reply:
OK no problem, let's keep them as a reminder. I'm not sure either what will happen with those functions/tests.